server/routes/create-an-activity/journey.ts

```diff
@@ -5,6 +5,12 @@ export type CreateAnActivityJourney = {
   }
   name?: string
   riskLevel?: string
-  minimumIncentive?: string
-  incentiveLevels?: string[]
+  pay?: Array<{
+    incentiveLevel: string
+    bandId: number
+    bandAlias: string
+    displaySequence: number
+    rate: number
+  }>
```

Comment:
Could this be a list type, or is there a reason for making defining the type inline so to speak?